core/src/main/res/values/strings.xml

```diff
@@ -447,17 +447,20 @@
     <string name="pref_theme_title_dark">Dark</string>
     <string name="pref_theme_title_trueblack">Black (AMOLED ready)</string>
     <string name="pref_episode_cache_unlimited">Unlimited</string>
-    <string name="pref_gpodnet_authenticate_title">Login</string>
-    <string name="pref_gpodnet_authenticate_sum">Login with your gpodder.net account in order to sync your subscriptions.</string>
-    <string name="pref_gpodnet_logout_title">Logout</string>
-    <string name="pref_gpodnet_logout_toast">Logout was successful</string>
+    <string name="pref_synchronization_logout_toast">Logout was successful</string>
     <string name="pref_gpodnet_setlogin_information_title">Change login information</string>
     <string name="pref_gpodnet_setlogin_information_sum">Change the login information for your gpodder.net account.</string>
-    <string name="pref_gpodnet_sync_changes_title">Synchronize now</string>
-    <string name="pref_gpodnet_sync_changes_sum">Sync subscription and episode state changes with gpodder.net.</string>
-    <string name="pref_gpodnet_full_sync_title">Force full synchronization</string>
-    <string name="pref_gpodnet_full_sync_sum">Sync all subscriptions and episode states with gpodder.net.</string>
-    <string name="pref_gpodnet_login_status"><![CDATA[Logged in as <i>%1$s</i> with device <i>%2$s</i>]]></string>
+    <string name="synchronization_sync_changes_title">Synchronize now</string>
+    <string name="synchronization_full_sync_title">Force full synchronization</string>
+    <string name="synchronization_login_status"><![CDATA[Logged in as <i>%1$s</i>. <br/><br/>%2$s]]></string>
+    <string name="synchronization_summary_unchoosen">You can choose from multiple providers to synchronize your subscriptions and episode play state with</string>
+    <string name="synchronization_summary_nextcloud">Gpoddersync is an open-source Nextcloud app that you can easily install on your own server. The app is independent of the AntennaPod project.</string>
+    <string name="synchronization_choose_and_login_summary">Choose a synchronization provider and login</string>
+    <string name="synchronization_choose_and_login_title">Choose synchronization provider</string>
+    <string name="synchronization_logout_summary">You can choose your synchronization provider again once you have logged out</string>
```

Comment:
I think `synchronization_login_status` and `synchronization_logout_summary` are always added together. They can simply be a single string then.

Reply:
merged